webapp/app/templates/basis/base_step_form.html

```diff
@@ -39,4 +39,50 @@
         $('input[type="date"]').attr('max', '9999-12-31');
     </script>
     <script src="{{ url_for('static', filename='js/jquery.mask.min.js') }}" type=text/javascript></script>
+    <script>
+            $(document).ready(function(){
+                $('.date_input').mask('00.00.0000');
+                $('input[name="unlock_code"]').mask('A', {
+                    translation: {
+                        "A": { pattern: /[\w]/, recursive: true }
+                    }
+                });
+                window.memo_values  = new Array();
+
+                 const sub_fields = document.querySelectorAll('input[name="unlock_code"]');
+                 sub_fields.forEach(input => {
+                     input.addEventListener('input', e => {
+                         console.log("old value" + window.memo_values[input.id]);
+                         console.log("Hallo" + input.value);
+                         const data = input.value;
+                         console.log(input.dataset.fieldLength);
+                         console.log(data.length, Number(input.dataset.fieldLength));
+                         console.log(data.length > Number(input.dataset.fieldLength));
+                         let field_to_focus = input
+                         if (data.length > Number(input.dataset.fieldLength)) {
+
+                             console.log("start splitting");
+                             let start_idx = 0;
+                             let start = false;
+                             sub_fields.forEach(sub_field => {
+                                 if (start || sub_field == input){
+                                     start = true;
+                                     const length = parseInt(sub_field.dataset.fieldLength);
+                                     const paste_content = data.substr(start_idx, length);
+                                     console.log("split content", paste_content, length);
+                                     if (paste_content.length > 0){
+                                        field_to_focus = sub_field;
+                                     };
+                                     sub_field.value = paste_content || '';
+                                     start_idx += length;
+                                 }
+                             });
+                             console.log(field_to_focus);
+                             field_to_focus.focus();
+                         }
```

Comment:
Is there a more specific place for this than `base_step_form`? There's no need to include this on every step form. (But maybe we don't have a template for unlock_code fields?)

Reply:
This script was added by mistake (s. below)